dashboard/test/controllers/api_controller_test.rb

```diff
@@ -1288,16 +1288,6 @@ def make_progress_in_section(script)
     )
   end
 
-  test 'should show teacher-dashboard link when a teacher' do
-    teacher = create :teacher
-    sign_in teacher
-
-    get :user_menu
-
-    assert_response :success
-    assert_select 'a[href="//test.code.org/teacher-dashboard"]', 'Teacher Home Page'
-  end
-
```

Comment:
In this case we just reduced our surface area.  If we wouldn't have written the test in the first place, then it doesn't seem to have it here now.  The only reason to write a test now is if we're worried about somehow regressing this removal, which seems unlikely!